doc/tutorial/fcm_fundamentals

```diff
@@ -0,0 +1,859 @@
+                        <a href="fcm_specifics#S3">Trac</a> tutorial for more on Trac).
+                    </li>
+                </ul>
+                <br/>
+                The Fcm repository is backed up so a personal back-up is not required.
+                <br/>
+                Any local repository that has been personally created will not necessarily be 
+                backed up so the responsiblility is on the person.
+                <br/>
+                <br/>
+                <h5>Creating your own central Fcm repository</h5>
+                Normally the Fcm team will help with setting up the initial repository.
+                <br/>
+                A personal repository can be created as follows:
+                <br/>
+                <br/>
+                First create a directory to work in on the Fcm tutorial material:
+                <br/>
+                Here we will be using commands that have not been fully explained yet.
+                They will all be covered in the Essentials tutorials.
+                <pre>
+cd
+mkdir ~/fcm_tutorial
+cd ~/fcm_tutorial
+svnadmin create repository
+</pre>
+                <br/>
+                This creates a repository called <samp>repository</samp> using the 
+                <code>svnadmin</code> tool.
+                <pre>
+fcm mkdir file:///$HOME/fcm_tutorial/repository/fundamentals -m 'New project fundamentals'
+</pre>
+                <br/>
+                using the <code>-m option</code> means that we do not have to add the comment 
+                to the editor which is involved by the command for the commit message.
+                <br/>
+                We can see from the command that we can use <code>mkdir</code> to make the 
+                directory like we would in Linux. Many of the commands that we can use are 
+                the same as we would use in Linux to help make it easier for the user.
+                <br/>
+                The returned message is:
+                <pre>
+<font color="blue">
+Committed revision 1.
+</font>
+</pre>
+                <br/>
+                <pre>
+fcm mkdir file:///$HOME/fcm_tutorial/repository/fundamentals/trunk -m 'New folder fundamentals/trunk'
+fcm mkdir file:///$HOME/fcm_tutorial/repository/fundamentals/branches -m 'New folder fundamentals/branches'
+</pre>
+                <br/>
+                The returned messages are:
+                <pre>
+<font color="blue">
+Committed revision 2.
+
+Committed revision 3.
+</font>
+</pre>
+                respectively.
+                <br/>
+                <pre>
+fcm ls file:///$HOME/fcm_tutorial/repository/fundamentals
+</pre>
+                This lists the changes to the repository and returns:
+                <pre>
+<font color="blue">
+branches/
+trunk/
+</font>
+</pre>
+                Now to make our working copy, this uses Linux <code>mkdir -p</code> to create 
+                the directory and the path to it if it does not already exist:
```

Comment:
Perhaps highlight after this that the working copy and the repository are not the same place?